optaplanner-8-benchmarks/optaplanner-perf-benchmark/src/main/java/org/jboss/qa/brms/performance/realworld/cloudbalancing/CloudBalanceBenchmark.java

```diff
@@ -24,6 +24,12 @@
     @Param({"CB_100_300", "CB_1600_4800", "CB_10000_30000"})
     private CloudBalancingExample.DataSet dataset;
 
+    public CloudBalanceBenchmark(){}
```

Comment:
Formatting (consistency):
```suggestion
    public CloudBalanceBenchmark() {
    }
```